### ISSUE TITLE
rssw-oe-main:eu.rssw.antlr.proparse.checks.EmptyBlock should also flag methods/procedures/functions that only contain return

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#649]
In my option a method with just a return is equally bad as an empty method.
When a method with only retrurn is required there should be a comment declaring why it is doing nothing.

I see this as an extension of the existing rule, possible with a parameter, or it could be a new rule where we can decide which rule to use to detect empty blocks.

### Noncompliant Code Examples
```
PROCEDURE test: 
    RETURN.
END.

METHOD PUBLIC VOID test:
    RETURN.
END.

METHOD PUBLIC CHARACTER test:
    DEFINE VARIABLE return AS CHARACTER NO-UNDO.
    RETURN return.
END.

```
### Compliant Solutions
```
PROCEDURE test: 
  /* Nothing done in this case */
  RETURN.
END PROCEDURE.

METHOD test:
    /* required by interface, Nothing done in this case */ 
    RETURN.
END.

METHOD PUBLIC CHARACTER test:
    DEFINE VARIABLE return AS CHARACTER NO-UNDO.
    /* always empty */
    RETURN return.
END.

METHOD PUBLIC CHARACTER test:
    RETURN "".
END.

```

**Comment by gquerret** [comment]
That would need a bit of improvement on the engine to know which statements are "action" statements, but that can definitely be considered in a next release.

**Comment by gquerret** [comment]
Added in develop branch
